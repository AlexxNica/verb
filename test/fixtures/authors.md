# Authors
{% authors().forEach(function(author) { %}
+ [{%= author.name %}]({%= author.url %}) {% }); %}