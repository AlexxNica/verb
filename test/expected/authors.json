[
  {
    "name": "Jon Schlinkert",
    "email": "",
    "url": "http://github.com/jonschlinkert"
  },
  {
    "name": "Brian Woodward",
    "email": "",
    "url": "http://github.com/doowb"
  }
]